src/helpers/ens.ts

```diff
@@ -94,6 +99,7 @@ export type TextRecordField = {
 
 export const textRecordFields = {
   [ENS_RECORDS.displayName]: {
+    defaultRecord: true,
```

Comment:
Wonder if this naming could get confused with having that record as a default selected field (and not a default field on the sheet). 

Maybe better naming could be as simple/direct as `isPrimaryDisplayRecord`?

Reply:
cool i'll change it